README.md

```diff
@@ -47,12 +47,55 @@ https://vercel.com/new/git/external
     ISSUER_BASE_URL,
     CLIENT_ID,
     APP_SESSION_SECRET,
+    NODE_ENV,
     VERCEL_URL
   &project-name=web-app-unsecured
   &repository-name=web-app-unsecured
 ```
 
-[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https%3A%2F%2Fgithub.com%2Fauth0%2Fauth0-product-education-labs%2Ftree%2Fvercel-web-application-unsecure%2Fapps%2Fweb-app-unsecured&env=ISSUER_BASE_URL,CLIENT_ID,APP_SESSION_SECRET,VERCEL_URL&project-name=web-app-unsecured&repository-name=web-app-unsecured)
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https%3A%2F%2Fgithub.com%2Fauth0%2Fauth0-product-education-labs%2Ftree%2Fvercel-web-application-unsecure%2Fapps%2Fweb-app-unsecured&env=ISSUER_BASE_URL,CLIENT_ID,APP_SESSION_SECRET,NODE_ENV,VERCEL_URL&project-name=web-app-unsecured&repository-name=web-app-unsecured)
+
+### Front End Web App - Secured
+
+The Secured Front End Web App is a simple express application that is intended to be used as a starting point for hands on labs.
+
+#### Environment Variables
+
+| Variable        | Value                                | Default |
+| --------------- | ------------------------------------ | ------- |
+| ISSUER_BASE_URL | https://your-tenant.region.auth0.com | ❌      |
+| CLIENT_ID       | application client id from Auth0     | ❌      |
+| API_URL         | url for expenses api                 | ❌      |
+| SECRET          | openssl rand -hex 32                 | ✅      |
+| PORT            | **7000**                             | ✅      |
+
+#### Run Local:
+
+```bash
+ISSUER_BASE_URL=https://your-tenant.region.auth0.com \
+CLIENT_ID=your-app-client-id  \
+API_URL=http://url-to-expenses-api \
+npm run front-end-web-app:secure:start
+```
+
+#### Vercel Deploy:
+
+```
+https://vercel.com/new/git/external
+  ?repository-url=https%3A%2F%2Fgithub.com%2Fauth0%2Fauth0-product-education-labs%2Ftree%2Fvercel-front-end-web-application-secure%2Fapps%2Ffront-end-web-app-secured
+  &env=
+    ISSUER_BASE_URL,
+    CLIENT_ID,
+    SECRET,
+    API_URL,
+    NODE_ENV,
+    VERCEL_URL,
+    VERCEL_GITHUB_REPO
+  &project-name=front-end-web-app-secured
+  &repository-name=front-end-web-app-secured
+```
+
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https%3A%2F%2Fgithub.com%2Fauth0%2Fauth0-product-education-labs%2Ftree%2Fvercel-front-end-web-application-secure%2Fapps%2Ffront-end-web-app-secured&env=ISSUER_BASE_URL,CLIENT_ID,SECRET,API_URL,NODE_ENV,VERCEL_URL,VERCEL_GITHUB_REPO&project-name=front-end-web-app-secured&repository-name=front-end-web-app-secured)
```

Comment:
Vercel requesting `NODE_ENV` as a Required Environment Variable. Is this `SECRET` I the table above?

Reply:
Ill update the docs to reflect this.